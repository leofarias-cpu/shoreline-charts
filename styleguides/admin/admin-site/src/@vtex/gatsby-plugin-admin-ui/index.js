export default {
  sizes: {
    header: '4.5rem',
    content: '90rem',
  },
  space: {
    content: '4.5rem',
  },
}
