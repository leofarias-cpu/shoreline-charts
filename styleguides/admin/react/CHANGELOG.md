# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.4.3](https://github.com/vtex/onda/compare/@vtex/onda-react@0.4.2...@vtex/onda-react@0.4.3) (2021-07-20)

**Note:** Version bump only for package @vtex/onda-react





## [0.4.2](https://github.com/vtex/onda/compare/@vtex/onda-react@0.4.1...@vtex/onda-react@0.4.2) (2021-07-19)

**Note:** Version bump only for package @vtex/onda-react





## [0.4.1](https://github.com/vtex/onda/compare/@vtex/onda-react@0.4.0...@vtex/onda-react@0.4.1) (2021-07-16)


### Bug Fixes

* **onda-react:** remove the cleaning of options properties from jsx ([b8ec31c](https://github.com/vtex/onda/commit/b8ec31cf1e557996ea8eb7027edbcfcbe55845fb))





# [0.4.0](https://github.com/vtex/onda/compare/@vtex/onda-react@0.3.0...@vtex/onda-react@0.4.0) (2021-07-15)


### Features

* **admin-ui/ultimate-table:** support sorting ([7453750](https://github.com/vtex/onda/commit/745375001bd28bb73e8a229f14c12e78bd0385da))





# [0.3.0](https://github.com/vtex/onda/compare/@vtex/onda-react@0.2.2...@vtex/onda-react@0.3.0) (2021-07-15)


### Bug Fixes

* **onda-react:** jsx refs type inference ([5a24609](https://github.com/vtex/onda/commit/5a2460960c419a272f16d76a43f9575247643f2a))
* **onda-react:** jsx rendering issues ([6fbe2b5](https://github.com/vtex/onda/commit/6fbe2b585ab083a99a1aa806de3ed0e23ac2451f))
* **onda-react:** typescript inference for jsx.elements ([fb9e365](https://github.com/vtex/onda/commit/fb9e365df6a877a274bc3ea26131a4015fbbcd33))


### Features

* **onda-react:** add ComponentProps type ([e13433c](https://github.com/vtex/onda/commit/e13433c2b9611de3cd6723541fe82064bd1f10cb))





## [0.2.2](https://github.com/vtex/onda/compare/@vtex/onda-react@0.2.1...@vtex/onda-react@0.2.2) (2021-07-05)

**Note:** Version bump only for package @vtex/onda-react





## [0.2.1](https://github.com/vtex/onda/compare/@vtex/onda-react@0.2.0...@vtex/onda-react@0.2.1) (2021-07-01)


### Bug Fixes

* **onda-core:** build imports ([9d30f9e](https://github.com/vtex/onda/commit/9d30f9e743d75b03995547e4d3052aa5a510e095))





# [0.2.0](https://github.com/vtex/onda/compare/@vtex/onda-react@0.1.1...@vtex/onda-react@0.2.0) (2021-07-01)


### Bug Fixes

* **onda-react:** remove unnecessary state ([1465e69](https://github.com/vtex/onda/commit/1465e695a6cb9be6ba3773d0acf51194a0713ea2))
* **onda-react:** resolve jsx hof types ([2e185ac](https://github.com/vtex/onda/commit/2e185ac2dd319f30d1eb4332a13a2299a38e06cb))


### Features

* **onda-react:** rephrase createComponent as jsx ([dd157ff](https://github.com/vtex/onda/commit/dd157ffa2500ed8e911b7d2d04db65268fc7f27b))





## [0.1.1](https://github.com/vtex/onda/compare/@vtex/onda-react@0.1.0-labs.4...@vtex/onda-react@0.1.1) (2021-06-29)


### Bug Fixes

* **onda-react:** package.json version ([fb98ac4](https://github.com/vtex/onda/commit/fb98ac411f80ae253225940a7c687cb458677e4a))





# 0.1.0-labs.4 (2021-06-24)


### Features

* **onda-packages:** add onda packages ([b619064](https://github.com/vtex/onda/commit/b619064d04d190d5615a2832dbcb86d96efd16ee))





# [0.1.0-labs.3](https://github.com/vtex/onda/compare/@vtex/onda-react@0.1.0-labs.2...@vtex/onda-react@0.1.0-labs.3) (2021-05-19)


### Features

* **onda:** allow global styles from theme ([aff3fa4](https://github.com/vtex/onda/commit/aff3fa449c6b92004312bf8ee4f648d7e6258dc0))
* **onda-core:** transform theme values on css variables ([3a8a379](https://github.com/vtex/onda/commit/3a8a379c3ccebf4738b65d39fc38aa8ac594e621))
* **onda-runtime-emotion:** allow default entries ([73a0dea](https://github.com/vtex/onda/commit/73a0deafda0b18ca09ca6eca00637d0c976055d8))





# [0.1.0-labs.2](https://github.com/vtex/onda/compare/@vtex/onda-react@0.1.0-labs.1...@vtex/onda-react@0.1.0-labs.2) (2021-05-13)


### Features

* **onda-react:** allow state property ([47bfb26](https://github.com/vtex/onda/commit/47bfb26bdb2f2f5496a9458505913f8133dfbee2))





# [0.1.0-labs.1](https://github.com/vtex/onda/compare/@vtex/onda-react@0.1.0-labs.0...@vtex/onda-react@0.1.0-labs.1) (2021-05-13)

**Note:** Version bump only for package @vtex/onda-react





# 0.1.0-labs.0 (2021-05-13)


### Features

* **onda:** add /react package ([4728b63](https://github.com/vtex/onda/commit/4728b631011945c4bd25a6e2de57c2d2e4f52c17))
