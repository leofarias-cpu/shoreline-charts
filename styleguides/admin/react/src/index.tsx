export * from './tag'
export * from './domElements'
export * from './jsx'
export * from './types'
export * from './useStyleSheet'
export * from './util'
export * from './symbols'
