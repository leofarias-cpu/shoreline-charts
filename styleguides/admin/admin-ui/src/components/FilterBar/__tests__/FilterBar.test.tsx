import React from 'react'
import { render } from '@testing-library/react'
import { ThemeProvider } from '@vtex/admin-core'
import { axe } from 'jest-axe'

import { FilterBar } from '../index'

describe('FilterBar tests', () => {
  beforeAll(() => {
    Object.defineProperty(window, 'matchMedia', {
      writable: true,
      value: jest.fn().mockImplementation((query) => ({
        matches: false,
        media: query,
        onchange: null,
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
        dispatchEvent: jest.fn(),
      })),
    })
  })

  it('should have overridable styles', () => {
    const { getByTestId } = render(
      <ThemeProvider>
        <FilterBar
          label="Filter Bar"
          csx={{ bg: 'coral' }}
          data-testid="filter-bar"
          onStatementChange={() => {}}
          filters={[
            {
              label: 'Filter',
              id: 'filter',
              conditions: [{ label: 'is', id: '1' }],
              resolver: {
                type: 'simple',
                defaultValue: { value: 'item 1' },
                items: [
                  { value: 'item 1' },
                  { value: 'item 2' },
                  { value: 'item 3' },
                ],
              },
            },
          ]}
        />
      </ThemeProvider>
    )

    expect(getByTestId('filter-bar')).toHaveStyleRule(
      'background-color',
      'coral'
    )
  })

  it('should match snapshot', async () => {
    const { asFragment } = render(
      <ThemeProvider>
        <FilterBar
          label="Filter Bar"
          csx={{ bg: 'coral' }}
          data-testid="filter-bar"
          onStatementChange={() => {}}
          statements={[
            {
              condition: { label: 'is', id: '1' },
              filter: {
                label: 'Filter',
                id: 'filter',
                conditions: [{ label: 'is', id: '1' }],
                resolver: {
                  type: 'simple',
                  defaultValue: { value: 'item 1' },
                  items: [
                    { value: 'item 1' },
                    { value: 'item 2' },
                    { value: 'item 3' },
                  ],
                },
              },
              target: { value: 'item 1' },
            },
          ]}
          filters={[
            {
              label: 'Filter',
              id: 'filter',
              conditions: [{ label: 'is', id: '1' }],
              resolver: {
                type: 'simple',
                defaultValue: { value: 'item 1' },
                items: [
                  { value: 'item 1' },
                  { value: 'item 2' },
                  { value: 'item 3' },
                ],
              },
            },
          ]}
        />
      </ThemeProvider>
    )

    expect(asFragment()).toMatchSnapshot()
  })

  it('should not have a11y violations', async () => {
    const { container } = render(
      <ThemeProvider>
        <FilterBar
          label="Filter Bar"
          csx={{ bg: 'coral' }}
          data-testid="filter-bar"
          onStatementChange={() => {}}
          statements={[
            {
              condition: { label: 'is', id: '1' },
              filter: {
                label: 'Filter',
                id: 'filter',
                conditions: [{ label: 'is', id: '1' }],
                resolver: {
                  type: 'simple',
                  defaultValue: { value: 'item 1' },
                  items: [
                    { value: 'item 1' },
                    { value: 'item 2' },
                    { value: 'item 3' },
                  ],
                },
              },
              target: { value: 'item 1' },
            },
          ]}
          filters={[
            {
              label: 'Filter',
              id: 'filter',
              conditions: [{ label: 'is', id: '1' }],
              resolver: {
                type: 'simple',
                defaultValue: { value: 'item 1' },
                items: [
                  { value: 'item 1' },
                  { value: 'item 2' },
                  { value: 'item 3' },
                ],
              },
            },
          ]}
        />
      </ThemeProvider>
    )

    const results = await axe(container)

    expect(results).toHaveNoViolations()
  })
})
