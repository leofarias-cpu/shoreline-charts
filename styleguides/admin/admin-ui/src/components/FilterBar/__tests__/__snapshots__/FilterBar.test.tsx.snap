// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterBar tests should match snapshot 1`] = `
<DocumentFragment>
  .emotion-39 {
  border-width: 1px;
  border-style: solid;
  border-radius: 4px;
  border-color: #E0E2E7;
  background-color: coral;
}

.emotion-38 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding-top: 0.5rem;
  padding-bottom: 0.5rem;
  padding-left: 0.75rem;
  padding-right: 0.75rem;
  border-top-width: 1px;
  border-top-style: solid;
  border-top-left-radius: 0;
  border-top-right-radius: 0;
  border-color: #E0E2E7;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  background-color: #FFFFFF;
}

.emotion-35 {
  font-family: VTEXTrustVF,-apple-system,system-ui,BlinkMacSystemFont,sans-serif;
  font-variation-settings: 'wght' 92;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  position: relative;
  background-color: transparent;
  color: #2953B2;
  font-size: 0.75rem;
  height: 32px;
  width: auto;
  padding-left: 0.5rem;
  padding-right: 0.75rem;
}

.emotion-35:focus:not([data-focus-visible-added]) {
  outline: none;
  box-shadow: none;
}

.emotion-35:focus {
  outline: none;
  box-shadow: 0rem 0rem 0rem 0.125rem #8DB6FA;
}

.emotion-35:hover {
  color: #214491;
  background-color: rgba(202,215,241,0.24);
}

.emotion-35:active {
  color: #1a3470;
  background-color: rgba(186,203,236,0.32);
}

.emotion-35:disabled {
  color: #898F9E;
}

.emotion-35 svg {
  width: 20px;
  min-width: 20px;
  height: 20px;
  min-height: 20px;
  margin: 0.25rem;
}

.emotion-20 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  width: 100%;
  margin: auto;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-19 {
  min-width: 24px;
  min-height: 24px;
  width: 24px;
  height: 24px;
}

.emotion-37 {
  font-family: VTEXTrustVF,-apple-system,system-ui,BlinkMacSystemFont,sans-serif;
  font-variation-settings: 'wght' 92;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  position: relative;
  color: #707685;
  background-color: transparent;
  font-size: 0.75rem;
  height: 32px;
  width: auto;
  padding-left: 0.75rem;
  padding-right: 0.75rem;
}

.emotion-37:focus:not([data-focus-visible-added]) {
  outline: none;
  box-shadow: none;
}

.emotion-37:focus {
  outline: none;
  box-shadow: 0rem 0rem 0rem 0.125rem #8DB6FA;
}

.emotion-37:hover {
  background-color: rgba(50,56,69,0.04);
}

.emotion-37:active {
  background-color: rgba(50,56,69,0.08);
}

.emotion-37:disabled {
  color: #898F9E;
}

.emotion-32 {
  padding: 0.5rem;
  padding-left: 1rem;
}

.emotion-31 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  -webkit-box-pack: unset;
  -ms-flex-pack: unset;
  -webkit-justify-content: unset;
  justify-content: unset;
}

.emotion-31 >*:not(:last-child) {
  margin-bottom: 0.5rem;
}

.emotion-30 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  width: 100%;
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: start;
  justify-content: start;
}

.emotion-18 >*:not(:first-child) {
  margin-left: 0.5rem;
}

.emotion-0 {
  padding-left: 0.75rem;
  min-width: 100px;
  max-width: 100px;
}

.emotion-6 {
  position: relative;
}

.emotion-4 {
  font-family: VTEXTrustVF,-apple-system,system-ui,BlinkMacSystemFont,sans-serif;
  font-variation-settings: 'wght' 92;
  border-width: 1px;
  border-style: solid;
  border-radius: 4px;
  border-color: #E0E2E7;
  cursor: pointer;
  position: relative;
  color: #707685;
  background-color: #FFFFFF;
  font-size: 0.875rem;
  height: 40px;
  width: auto;
  padding-left: 0.75rem;
  padding-right: 0.5rem;
  min-width: 150px;
  max-width: 150px;
}

.emotion-4:focus:not([data-focus-visible-added]) {
  outline: none;
  box-shadow: none;
}

.emotion-4:focus {
  outline: none;
  box-shadow: 0rem 0rem 0rem 0.125rem #8DB6FA;
}

.emotion-4:hover {
  background-color: rgba(50,56,69,0.04);
}

.emotion-4:active {
  background-color: rgba(50,56,69,0.08);
}

.emotion-4:disabled {
  color: #898F9E;
}

.emotion-4 svg {
  width: 24px;
  min-width: 24px;
  height: 24px;
  min-height: 24px;
  margin: 0.25rem;
}

.emotion-4 div {
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  width: 100%;
  margin: auto;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex-direction: row-reverse;
  -ms-flex-direction: row-reverse;
  flex-direction: row-reverse;
}

.emotion-1 {
  min-width: 24px;
  min-height: 24px;
  width: 24px;
  height: 24px;
  -webkit-transform: rotate(180deg);
  -moz-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
}

.emotion-2 {
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
}

.emotion-5 {
  visibility: hidden;
  cursor: pointer;
  background-color: #FFFFFF;
  border-radius: 4px;
  border-width: 1px;
  border-style: solid;
  border-color: #C4C5CA;
  box-shadow: 0rem 0rem 1rem 0rem rgba(0,0,0,0.10);
  outline: none;
  margin-top: 0.25rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
  min-width: inherit;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  position: absolute;
  z-index: 999;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: unset;
  -webkit-box-align: unset;
  -ms-flex-align: unset;
  align-items: unset;
  -webkit-box-pack: unset;
  -ms-flex-pack: unset;
  -webkit-justify-content: unset;
  justify-content: unset;
}

.emotion-5 >*:not(:last-child) {
  margin-bottom: 0.5rem;
}

.emotion-21 {
  font-family: VTEXTrustVF,-apple-system,system-ui,BlinkMacSystemFont,sans-serif;
  font-variation-settings: 'wght' 92;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  position: relative;
  color: #707685;
  background-color: transparent;
  font-size: 0.875rem;
  height: 40px;
  width: 40px;
  padding-left: 0.25rem;
  padding-right: 0.25rem;
}

.emotion-21:focus:not([data-focus-visible-added]) {
  outline: none;
  box-shadow: none;
}

.emotion-21:focus {
  outline: none;
  box-shadow: 0rem 0rem 0rem 0.125rem #8DB6FA;
}

.emotion-21:hover {
  background-color: rgba(50,56,69,0.04);
}

.emotion-21:active {
  background-color: rgba(50,56,69,0.08);
}

.emotion-21:disabled {
  color: #898F9E;
}

.emotion-21 svg {
  width: 24px;
  min-width: 24px;
  height: 24px;
  min-height: 24px;
  margin: 0.25rem;
}

.emotion-29 {
  border: 0;
  padding: 0rem;
  outline: none;
  z-index: 999;
}

.emotion-28 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: #FFFFFF;
  margin-top: 0.25rem;
  padding: 0.75rem;
  min-width: 18px;
  border-radius: 3px;
  border-width: 1px;
  border-style: solid;
  border-color: #C4C5CA;
  box-shadow: 0rem 0rem 1rem 0rem rgba(0,0,0,0.10);
}

.emotion-28 hr {
  margin-top: 0.5rem;
  margin-bottom: 0.5rem;
  border-style: solid;
  border-bottom-width: 1px;
  border-top: none;
  border-left: none;
  border-right: none;
  border-color: #C4C5CA;
  width: calc(100% -0.75rem);
  margin-left: -0.75rem;
  margin-right: -0.75rem;
  outline: none;
}

.emotion-28 item {
  margin-top: 0.125rem;
  margin-bottom: 0.125rem;
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  font-size: 0.875rem;
  border: none;
  text-transform: initial;
  width: 100%;
  color: #323845;
}

.emotion-28 item div {
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
}

.emotion-28 item svg {
  margin-left: 0rem;
  margin-right: 0.5rem;
}

.emotion-28 item:focus {
  background-color: rgba(218,227,245,0.32);
  outline: none;
  box-shadow: none;
}

.emotion-28 item:hover {
  color: #323845;
}

.emotion-28 item-dangerous {
  margin-top: 0.125rem;
  margin-bottom: 0.125rem;
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  font-size: 0.875rem;
  border: none;
  text-transform: initial;
  width: 100%;
  color: #CB4242;
}

.emotion-28 item-dangerous div {
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
}

.emotion-28 item-dangerous svg {
  margin-left: 0rem;
  margin-right: 0.5rem;
}

.emotion-28 item-dangerous:focus {
  background-color: rgba(254,218,218,0.32);
  outline: none;
  box-shadow: none;
}

.emotion-28 item-dangerous:hover {
  color: #CB4242;
}

.emotion-24 {
  font-family: VTEXTrustVF,-apple-system,system-ui,BlinkMacSystemFont,sans-serif;
  font-variation-settings: 'wght' 92;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  position: relative;
  background-color: transparent;
  color: #323845;
  font-size: 0.875rem;
  height: 32px;
  width: 100%;
  padding-left: 0.25rem;
  padding-right: 0.25rem;
  margin-top: 0.125rem;
  margin-bottom: 0.125rem;
  text-transform: initial;
}

.emotion-24:focus:not([data-focus-visible-added]) {
  outline: none;
  box-shadow: none;
}

.emotion-24:focus {
  background-color: rgba(218,227,245,0.32);
  outline: none;
  box-shadow: none;
}

.emotion-24:hover {
  color: #323845;
}

.emotion-24:active {
  color: #1a3470;
  background-color: rgba(186,203,236,0.32);
}

.emotion-24:disabled {
  color: #898F9E;
}

.emotion-24 svg {
  margin-left: 0rem;
  margin-right: 0.5rem;
}

.emotion-24 div {
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
}

.emotion-22 {
  min-width: 20px;
  min-height: 20px;
  width: 20px;
  height: 20px;
}

<div
    class="emotion-39"
    data-testid="filter-bar"
  >
    <div
      class="emotion-32"
    >
      <div
        class="emotion-31"
      >
        <div
          class="emotion-30"
        >
          <div
            class="emotion-18"
          >
            <div
              class="emotion-0"
            >
              Where
            </div>
            <div
              class="emotion-6"
            >
              <span
                style="border: 0px; height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;"
              >
                <label
                  for="downshift-0-toggle-button"
                  id="downshift-0-label"
                >
                  Filter
                </label>
              </span>
              <button
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-labelledby="downshift-0-label downshift-0-toggle-button"
                class="emotion-4"
                id="downshift-0-toggle-button"
                type="button"
              >
                <div
                  class="emotion-3"
                >
                  <svg
                    class="emotion-1"
                    fill="none"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16 14L12 10L8 14"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.5"
                    />
                  </svg>
                  <div
                    class="emotion-2"
                  >
                    Filter
                  </div>
                </div>
              </button>
              <div
                aria-labelledby="downshift-0-label"
                class="emotion-5"
                id="downshift-0-menu"
                role="listbox"
                tabindex="-1"
              />
            </div>
            <div
              class="emotion-6"
            >
              <span
                style="border: 0px; height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;"
              >
                <label
                  for="downshift-1-toggle-button"
                  id="downshift-1-label"
                >
                  Condition
                </label>
              </span>
              <button
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-labelledby="downshift-1-label downshift-1-toggle-button"
                class="emotion-4"
                id="downshift-1-toggle-button"
                type="button"
              >
                <div
                  class="emotion-3"
                >
                  <svg
                    class="emotion-1"
                    fill="none"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16 14L12 10L8 14"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.5"
                    />
                  </svg>
                  <div
                    class="emotion-2"
                  >
                    is
                  </div>
                </div>
              </button>
              <div
                aria-labelledby="downshift-1-label"
                class="emotion-5"
                id="downshift-1-menu"
                role="listbox"
                tabindex="-1"
              />
            </div>
            <div
              class="emotion-6"
            >
              <span
                style="border: 0px; height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;"
              >
                <label
                  for="downshift-2-toggle-button"
                  id="downshift-2-label"
                >
                  Value
                </label>
              </span>
              <button
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-labelledby="downshift-2-label downshift-2-toggle-button"
                class="emotion-4"
                id="downshift-2-toggle-button"
                type="button"
              >
                <div
                  class="emotion-3"
                >
                  <svg
                    class="emotion-1"
                    fill="none"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16 14L12 10L8 14"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.5"
                    />
                  </svg>
                  item 1
                </div>
              </button>
              <div
                aria-labelledby="downshift-2-label"
                class="emotion-5"
                id="downshift-2-menu"
                role="listbox"
                tabindex="-1"
              />
            </div>
          </div>
          <button
            aria-controls="statement-menu"
            aria-expanded="false"
            aria-haspopup="menu"
            aria-label="statement-menu-0"
            class="emotion-21"
            type="button"
          >
            <div
              class="emotion-20"
            >
              <svg
                class="emotion-19"
                fill="none"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.25 12.004C11.25 12.418 11.586 12.754 12 12.75C12.414 12.75 12.75 12.414 12.75 12C12.75 11.586 12.414 11.25 12 11.25C11.586 11.252 11.25 11.588 11.25 12.004"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
                <path
                  d="M11.25 6.00398C11.25 6.41798 11.586 6.75398 12 6.74998C12.414 6.74998 12.75 6.41398 12.75 5.99998C12.75 5.58598 12.414 5.24998 12 5.24998C11.586 5.25198 11.25 5.58798 11.25 6.00398"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
                <path
                  d="M11.25 18.004C11.25 18.418 11.586 18.754 12 18.75C12.414 18.75 12.75 18.414 12.75 18C12.75 17.586 12.414 17.25 12 17.25C11.586 17.252 11.25 17.588 11.25 18.004"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </div>
          </button>
          <div
            aria-label="statement-menu-0"
            aria-orientation="vertical"
            class="emotion-29"
            data-dialog="true"
            hidden=""
            id="statement-menu"
            role="menu"
            style="display: none;"
            tabindex="-1"
          >
            <div
              class="emotion-28"
            >
              <button
                class="emotion-24"
                id="statement-menu-1"
                role="menuitem"
                tabindex="0"
                type="button"
              >
                <div
                  class="emotion-20"
                >
                  <svg
                    class="emotion-22"
                    fill="none"
                    viewBox="0 0 24 24"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M4.44852 3.75488C4.06573 3.75488 3.75452 4.0661 3.75452 4.44888V14.5609C3.75452 14.9437 4.06573 15.2549 4.44852 15.2549H5.00452C5.41873 15.2549 5.75452 15.5907 5.75452 16.0049C5.75452 16.4191 5.41873 16.7549 5.00452 16.7549H4.44852C3.2373 16.7549 2.25452 15.7721 2.25452 14.5609V4.44888C2.25452 3.23767 3.2373 2.25488 4.44852 2.25488H14.5595C15.7714 2.25488 16.7545 3.23733 16.7545 4.44888V5.00488C16.7545 5.4191 16.4187 5.75488 16.0045 5.75488C15.5903 5.75488 15.2545 5.4191 15.2545 5.00488V4.44888C15.2545 4.06643 14.9436 3.75488 14.5595 3.75488H4.44852Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                    <path
                      clip-rule="evenodd"
                      d="M9.44852 8.75488C9.06573 8.75488 8.75452 9.0661 8.75452 9.44888V19.5609C8.75452 19.9437 9.06573 20.2549 9.44852 20.2549H19.5605C19.943 20.2549 20.2545 19.944 20.2545 19.5599V9.44888C20.2545 9.06643 19.9436 8.75488 19.5595 8.75488H9.44852ZM7.25452 9.44888C7.25452 8.23767 8.2373 7.25488 9.44852 7.25488H19.5595C20.7714 7.25488 21.7545 8.23733 21.7545 9.44888V19.5599C21.7545 20.7718 20.7721 21.7549 19.5605 21.7549H9.44852C8.2373 21.7549 7.25452 20.7721 7.25452 19.5609V9.44888Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                    <path
                      clip-rule="evenodd"
                      d="M14.5045 11.2549C14.9187 11.2549 15.2545 11.5907 15.2545 12.0049V17.0049C15.2545 17.4191 14.9187 17.7549 14.5045 17.7549C14.0903 17.7549 13.7545 17.4191 13.7545 17.0049V12.0049C13.7545 11.5907 14.0903 11.2549 14.5045 11.2549Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                    <path
                      clip-rule="evenodd"
                      d="M11.2545 14.5049C11.2545 14.0907 11.5903 13.7549 12.0045 13.7549H17.0045C17.4187 13.7549 17.7545 14.0907 17.7545 14.5049C17.7545 14.9191 17.4187 15.2549 17.0045 15.2549H12.0045C11.5903 15.2549 11.2545 14.9191 11.2545 14.5049Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                  </svg>
                  Duplicate
                </div>
              </button>
              <button
                class="emotion-24"
                id="statement-menu-2"
                role="menuitem"
                tabindex="-1"
                type="button"
              >
                <div
                  class="emotion-20"
                >
                  <svg
                    class="emotion-22"
                    fill="none"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M18 6V18.75C18 19.993 16.973 21 15.731 21H8.231C6.988 21 6 19.993 6 18.75V6"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1"
                    />
                    <path
                      d="M19.5 6H4.5"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1"
                    />
                    <path
                      d="M10 3H14"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1"
                    />
                    <path
                      d="M14 10V17"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1"
                    />
                    <path
                      d="M10 17V10"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1"
                    />
                  </svg>
                  Delete
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="emotion-38"
    >
      <button
        class="emotion-35"
        type="button"
      >
        <div
          class="emotion-20"
        >
          <svg
            class="emotion-19"
            fill="none"
            viewBox="0 0 24 24"
          >
            <path
              clip-rule="evenodd"
              d="M12.0045 5.59557C12.4187 5.59557 12.7545 5.93136 12.7545 6.34557L12.7545 17.6593C12.7545 18.0735 12.4187 18.4093 12.0045 18.4093C11.5903 18.4093 11.2545 18.0735 11.2545 17.6593L11.2545 6.34557C11.2545 5.93136 11.5903 5.59557 12.0045 5.59557Z"
              fill="currentColor"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M18.4114 12.0024C18.4114 12.4166 18.0756 12.7524 17.6614 12.7524H6.34766C5.93345 12.7524 5.59766 12.4166 5.59766 12.0024C5.59766 11.5882 5.93345 11.2524 6.34766 11.2524H17.6614C18.0756 11.2524 18.4114 11.5882 18.4114 12.0024Z"
              fill="currentColor"
              fill-rule="evenodd"
            />
          </svg>
          Add Filter
        </div>
      </button>
      <button
        class="emotion-37"
        type="button"
      >
        <div
          class="emotion-20"
        >
          Reset Filters
        </div>
      </button>
    </div>
  </div>
</DocumentFragment>
`;
